doc/src/features/Nachwuchskraefte.md

```diff
@@ -11,15 +11,21 @@ Displayed Infos:
 - Major/training direction
 - year
 
-### Modify [Nwks](./../glossary.md#nwk)
+## Modify [Nwks](./../glossary.md#nwk)
 
 Under each [Nwk](./../glossary.md#nwk) in the overview is a button labeled `Bearbeiten`.
+
 Clicking it will open a dialog with the [Nwk](./../glossary.md#nwk)'s information.
+
 It is possible to change the name, year and major/training direction.
+
 The changes will be saved after clicking the `Akzeptieren` button, and a green success message will appear.
+
 If the changes are not accepted, the old values will remain, and a red error message will appear.
 
-### Using the Import button
+Clicking on the `Abbrechen` button cancels the operation and nothing will be changed and the dialog will be closed.
+
+## Using the Import button
```

Comment:
`The Excel file has to be in this correct format.` Maybe we should make it clear that the link leads to an example file. Most of the other links lead to different pages.